doc/articles/features/android-tv.md

```diff
@@ -0,0 +1,77 @@
+---
+uid: Uno.Features.AndroidTv
+---
+
+# Support for Android TV
+
+Uno Platform is proud to support Android TV, enabling you to extend your application's reach to this wide family of devices with unique use cases.
+
+## Enabling Android TV support
+
+To make your application properly show up among installed apps on Android TV and to be able to publish the app to the store, you need to adjust the app manifest to declare support for Android TV.
+
+First, open the `MainActivity` class in the `YourApp.Mobile` project (or the `YourApp.Droid` project for older versions of Uno Platform), and add an `[IntentFilter]` attribute for `ActionMain` declaring support for `LeanbackLauncher` category:
```

Comment:
Are there downsides of including this by default in the templates?

Reply:
I would vote against it, because most apps will not be suitable for Android TV, so I would keep this as an optional thing.